src/formatters/diffs/object.js

```diff
@@ -0,0 +1,19 @@
+import makeDatesDiff from './date';
+import makeStringsDiff from './string';
+import truncateDiff from './truncate-diff';
+
+export default function makeObjectsDiff (err, maxStringLength) {
+    var diffs = {
+        expected: err.expected,
+        actual:   err.actual,
+        marker:   ''
+    };
+
+    if (err.diffType && err.diffType.isDates)
+        diffs = makeDatesDiff(err);
+    else if (err.diffType && err.diffType.isStrings)
+        diffs = makeStringsDiff(diffs, maxStringLength);
+    else diffs = truncateDiff(diffs, maxStringLength);
```

Comment:
I bet I saw this code in `array` diff, so seems like array just using `object` diff?